### DISCUSSION TITLE
Update gutter instantly when file is saved?

### DISCUSSION THREAD

**Discussion by Louis-dM** [airblade/vim-gitgutter#819] [Q&A] [answered]
Is it possible to update the gutter whenever the current file is saved? I don't mind the delay of a few seconds caused by `updatetime` for unsaved changes, but when I save I often immediately want to see what's been changed.

(I use `updatetime` for other things so I can't change it to a smaller value)

**Comment by airblade** [comment, marked as answer]
This is documented in the FAQ (see `:help gitgutter-faq` or read the [README](https://github.com/airblade/vim-gitgutter#faq)).

```viml
autocmd BufWritePost * GitGutter
```

**Reply by Louis-dM** [reply]
Well I thought I'd read the entire readme, but I completely missed that. Sorry, thanks for the info!
